dora/build.rs

```diff
@@ -34,10 +35,15 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     writeln!(&mut stdlib, r#"["#,)?;
 
-    for f in fs::read_dir(SOURCE_DIR)? {
+    for f in WalkDir::new(SOURCE_DIR) {
         let f = f?;
 
-        if !f.file_type()?.is_file() {
+        if f.file_type().is_dir() {
+            create_dir_all(out_path.join(f.path()))?;
+            continue;
+        }
+
+        if f.file_type().is_symlink() {
```

Comment:
Why do we need to handle symlinks explicitly?

Reply:
It's the preserved logic from before: !is_file --> is_dir || is_symlink

Reply:
Couldn't we just keep the current logic and just create the path before copying the file? That would have the advantage that we don't create directories without dora files.

Reply:
Yes, absolutely, thanks for the review, I changed it and it's much better that way!